i18n.py

```diff
@@ -1,15 +1,57 @@
+import io
 import os
+import shutil
 import subprocess
+import sys
+import zipfile
 
+import requests
 
 this_dir = os.path.dirname(os.path.abspath(__file__))
 locale_dir = os.path.join(this_dir, 'locale')
 
+msgfmt_path = '/Library/Frameworks/Python.framework/Versions/3.6/share/doc/python3.6/examples/Tools/i18n/msgfmt.py'
```

Comment:
does this make some strong assumptions about how python is installed that won't always be true?

Reply:
The main thing is that, for now, I've hardcoded the Python version. I can switch to getting the Python version programmatically to address that. 

Using other methods of installing Python will likely be problematic, since we are packaging the app for distribution. e.g. if we install with brew, it may depend on brew libraries not on the user's system. So I don't envision us changing the way we install Python in the foreseeable future.

Reply:
Just pushed a change that makes the code version-agnostic.